LiteDB/Engine/Disk/DiskWriterQueue.cs

```diff
@@ -37,9 +38,7 @@ public DiskWriterQueue(Stream stream)
             _waiter = new ManualResetEventSlim(false);
             _writing = new ManualResetEventSlim(false);
 
-            _thread = new Thread(this.CreateThread);
-            _thread.Name = "LiteDB_Writer";
-            _thread.Start();
+            _task = Task.Factory.StartNew(this.CreateThread, TaskCreationOptions.LongRunning);
```

Comment:
I am not sure, if it will work same as Thread. As minimum, we lose Thread.Name here.
But we always can use #if.